cluster/gce/config-default.sh

```diff
@@ -198,6 +198,12 @@ export MASTER_NODE_LABELS="${KUBE_MASTER_NODE_LABELS:-}"
 NON_MASTER_NODE_LABELS="${KUBE_NON_MASTER_NODE_LABELS:-}"
 WINDOWS_NON_MASTER_NODE_LABELS="${WINDOWS_NON_MASTER_NODE_LABELS:-}"
 
+# Network Policy plugin specific settings.
+NETWORK_POLICY_PROVIDER="${NETWORK_POLICY_PROVIDER:-none}" # calico
+
+# Windows Network Policy plugin specific settings.
+export c="${WINDOWS_NETWORK_POLICY_PROVIDER:-none}"  # antrea
```

Comment:
c? typo? I don't see this used later and it seems like a bad variable name.

Reply:
Good catch! Fixed it.